ci.js

```diff
@@ -0,0 +1,181 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+    console.log(
+        `Usage: "ci check-consistency" or "ci bump-version %postfix%" or "ci get-version"`
+    );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+switch (mode) {
+    case "get-version":
+        break;
```

Comment:
do I understand correctly that you do nothing in this case? Why to have this case then? if you want to add code to this in the future I think it would be good to have a TODO with the name of the task

Reply:
oh I see it is used again in run()
Maybe it would be better to just replace this switch with if (mode === "bump-version")
What do you think? Do we really need default case?